.github/workflows/scheduled_releases.yml

```diff
@@ -27,9 +27,7 @@ jobs:
        # we support the trailing four versions plus the latest pre-release
        # and the upcoming version's pre-release.
        version: ["1.1.0", "1.2.0", "1.3.0", "1.4.0", "1.5.0", "1.6.0b2"]
```

Comment:
Has the bug with re-releasing a pre-release of the same name been fixed?  If not we should remove the pre-release from this list.

Reply:
Will wait on merging this to validate #51